tools/mo/openvino/tools/mo/back/ie_ir_ver_2/emitter.py

```diff
@@ -641,7 +641,9 @@ def append_ir_info(file: str,
                    mean_data: [list, None] = None,
                    input_names: list = None,
                    legacy_path: bool = True):
-    path_to_xml = file + ".xml"
+    path_to_xml = file
+    if not path_to_xml.endswith(".xml"):
+        path_to_xml = file + ".xml"
```

Comment:
can we do not to change this code please? I would rather correct the code caller. What if the original model name contain `.xml` in its name?

Reply:
Done.